README.md

```diff
@@ -253,14 +253,15 @@ usdt = { version = "*", optional = true, default-features = false }
 usdt-probes = ["usdt/asm"]
 ```
 
-This allows users to opt into probes if they're willing to accept a nightly toolchain.
+This allows users to opt into probes if they are using a suitably new toolchain, or and older
+nightly with the `asm` feature enabled on their project.
 
 ### The Rust `asm` feature
 
-Recall from the example that the `usdt` crate relies on inline `asm`, which is [not yet][asm-issue] a
-stable Rust feature. This means that the code _calling_ the generated probe macros must
-be in a module where the `asm!` macro can be used, i.e., where the `feature(asm)` configuration
-directive is applicable.
+On toolchains prior to 1.59, inline asm was [not available][asm-issue] without the feature being
+enabled.  This applies to code _calling_ the probe macros, in addition to `usdt` where they are
+implemented.  Those generated probe macros must be in a module that is either built with a >=1.59
```

Comment:
I'd be explicit here: ">= 1.59 stable toolchain or a nightly toolchain where the `feature(asm)`..."

Reply:
Actually, I think there should be a note here about macOS. Something pointing out that macOS still requires nightly, due to the `asm_sym` and `asm_const` features not yet on stable.